suite-native/module-accounts-import/src/screens/XpubScanScreen.tsx

```diff
@@ -40,30 +42,42 @@ const chipStyle = prepareNativeStyle(utils => ({
 }));
 
 const devXpubButtonStyle = prepareNativeStyle(utils => ({
-    marginTop: utils.spacings.small,
+    marginTop: utils.spacings.large,
     borderRadius: utils.borders.radii.round,
 }));
 
-const DEFAULT_XPUB_INPUT_TEXT = '';
 const DEFAULT_CURRENCY_SYMBOL = 'btc';
 // Slip 0014 - https://github.com/satoshilabs/slips/blob/master/slip-0014.md#bitcoin-segwit--p2wpkh--bip84
 const BTC_HARD_CODED_XPUB =
     'zpub6rszzdAK6RuafeRwyN8z1cgWcXCuKbLmjjfnrW4fWKtcoXQ8787214pNJjnBG5UATyghuNzjn6Lfp5k5xymrLFJnCy46bMYJPyZsbpFGagT';
 
+const xpubFormValidationSchema = yup.object({
+    xpubAddress: yup.string().required(),
+});
+type XpubFormValues = yup.InferType<typeof xpubFormValidationSchema>;
+
 export const XpubScanScreen = ({
     navigation,
 }: StackProps<AccountsImportStackParamList, AccountsImportStackRoutes.XpubScan>) => {
     const [selectedCurrencySymbol, setSelectedCurrencySymbol] =
         useState<NetworkSymbol>(DEFAULT_CURRENCY_SYMBOL);
-    const [inputText, setInputText] = useState<string>(DEFAULT_XPUB_INPUT_TEXT);
     const [cameraRequested, setCameraRequested] = useState<boolean>(false);
     const { applyStyle } = useNativeStyles();
 
+    const form = useForm<XpubFormValues>({
+        validation: xpubFormValidationSchema,
+        defaultValues: {
+            // @TODO fix before merge, defaultValues not works...
+            xpubAddress: 'BlahBlah',
+        },
+    });
+    const { setValue, handleSubmit } = form;
+
     const resetToDefaultValues = useCallback(() => {
         setSelectedCurrencySymbol(DEFAULT_CURRENCY_SYMBOL);
-        setInputText(DEFAULT_XPUB_INPUT_TEXT);
+        setValue('xpubAddress', '');
```

Comment:
why not use reset functionality provided by RHF?

Reply:
> why not use reset functionality provided by RHF?

The simplest rule of thumb is:

- reset is for resting the whole form or it's major part
- setValue is for updating a single field value

https://github.com/react-hook-form/react-hook-form/discussions/7671#discussioncomment-2052400

Reply:
It's already removed anyway.